lib/rummager_republisher.rb

```diff
@@ -0,0 +1,48 @@
+class RummagerRepublisher
+  def republish_all
+    document_types.each do |document_type|
+      pagination = publishing_api_pagination_for(document_type)
+
+      puts "skipping - could not find #{document_type}" if pagination[:total_results].zero?
+      next if pagination[:total_results].zero?
+
+      while pagination[:current_page] <= pagination[:total_pages]
```

Comment:
I don't get what's going on here. 

`pagination[:current_page]` is not being mutated, and the inside of this block is  always the same?